viewer/viewerUtils.js

```diff
@@ -273,8 +273,10 @@ module.exports = (Config, Db, molochparser, internals) => {
 
       const indices = await Db.getIndices(reqQuery.startTime, reqQuery.stopTime, reqQuery.bounding, Config.get('rotateIndex', 'daily'));
       if (indices.length > 3000) { // Will url be too long
+        req._molochESQuery = query;
```

Comment:
Do you think this is the best place to do this? It's the last step buildSessionQuery. Alternatively, I could do it in the finish callback every time buildSessionQuery is called (~14 times).